libc/src/unistd/CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ add_entrypoint_object(
     .${LIBC_TARGET_OS}.pathconf
 )
 
-add_header_library(
+add_entrypoint_object(
```

Comment:
add_object_library

Reply:
Is there documentation that explains what is an object library ?

Reply:
To my understanding
- header library: header-only (`.h` files), does not generate object file
- object library: source files (`.cpp` files), generate object files
- entrypoint: similar to object library but it is used to expose symbols to userspace

Here you have a common library that has its own translation units (so object files are generated), and it is used in internal usage, not to expose symbols. So, object library would be proper.